### PR TITLE
created review ui and integrated with backend

### PR DESCRIPTION
**Pull request by AKILBEK-UR**

### CODE REVIEW

File: src/modules/home/api/get-booking-record.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { api } from '#/configs/axios.config';
+import { type User } from '#/types';
+import { useQuery } from '@tanstack/react-query';
+import { ENDPOINTS } from '../constants/endpoints.constant';
+import { type Accommodation } from '../types/accommodation.type';
+
+export type Booking = {
```
Comment (farrooh) [inline src/modules/home/api/get-booking-record.ts]: Maybe a separate file for types?

File: src/modules/home/components/single-property/single-property-rating.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+export function ReviewForm({ accommodationId }: ReviewFormProps) {
+  const [rating, setRating] = useState<number | null>(null);
+  const [comment, setComment] = useState<string>('');
+  const [error, setError] = useState<string>('');
+
+  const { mutate: createReview, status } = useCreateReview();
+  const isLoading = status === 'pending';
+  const handleSubmit = () => {
+    if (rating === null || comment.trim() === '') {
+      setError('Both rating and comment are required.');
```
Comment (farrooh) [inline src/modules/home/components/single-property/single-property-rating.tsx:22]: Comment should be optional.

**Review by farrooh — CHANGES_REQUESTED** [review]

File: src/modules/home/components/single-property/single-property-rating.tsx (hunk 2 above)
Comment (AKILBEK-UR) [reply src/modules/home/components/single-property/single-property-rating.tsx]: In our entity it already set nullable = false and have migration, should I change this?

Comment (farrooh) [reply src/modules/home/components/single-property/single-property-rating.tsx]: No, we can keep it like this then.

**Review by farrooh — APPROVED** [review]

**Review by purple-mountain — COMMENTED** [review]
why does this has other's changes

**Pull request closed without merging**